### ISSUE TITLE
Class imbalance

### ISSUE THREAD

**Issue by APJansen** [UNSAT3D/unsat#11]
The 5 classes are very imbalanced, occurring roughly with relative frequency (after throwing away the top and bottom 200 slices):

| class | occurrence (%) |
| -- | -- |
| water | 30 |
| background | 26 |
| air | 4 |
| root | 0.1 |
| soil | 38 |

This causes the model to basically ignore the root, and to a lesser extent the air.

I've solved this by computing the relative frequencies of the classes in the training set, and using the inverse of that as weights in the loss function. 
This results in a more even error across classes, but we should discuss if that's what we want.

**Comment by APJansen** [comment]
This is implemented in the loss function.
